core-experimental/include/iDynTree/Core/Rotation.h

```diff
@@ -42,7 +42,12 @@ namespace iDynTree
     {
     private:
         RotationSemantics semantics;
-
+        
+        /**
+         * Copy constructor: create a Rotation from another RotationRaw and another RotationSemantics.
+         */
+        Rotation(const RotationRaw & other, RotationSemantics & semantics);
```

Comment:
It was initially `private` I think, and I left it this way since I always protect attributes as much as possible. But here, we should use `protected` otherwise any derivate class won't have access to this attribute.